dockers/docker-fpm-quagga/bgpd.conf.j2

```diff
@@ -27,7 +27,7 @@ router bgp {{ DEVICE_METADATA['localhost']['bgp_asn'] }}
   bgp log-neighbor-changes
   bgp bestpath as-path multipath-relax
   no bgp default ipv4-unicast
-  bgp graceful-restart restart-time 180
+  bgp graceful-restart restart-time 240
```

Comment:
As per your description of the PR, it seems we are accommodating the value for some particular HW. I  would think it is better to make it as a configurable item in minigraph/configDB.  If not configured, the default is still 180, but on some particular platforms, we can increase the timer value.